lib/marqeta/transaction.rb

```diff
@@ -0,0 +1,29 @@
+module Marqeta
+  class Transaction < ApiObject
+    PENDING_STATE = 'PENDING'.freeze
+
+    def self.endpoint
+      'simulate/authorization'
+    end
+
+    def self.since(start_time)
+      params_string = URI.encode_www_form(
```

Comment:
Should this be handled by ApiCaller? i.e. you just pass it a hash, and it takes care of the encoding?